packages/plugin-doc-coverage/README.md

```diff
@@ -0,0 +1,204 @@
+# @code-pushup/doc-coverage-plugin
+
+[![npm](https://img.shields.io/npm/v/%40code-pushup%2Fdoc-coverage-plugin.svg)](https://www.npmjs.com/package/@code-pushup/doc-coverage-plugin)
+[![downloads](https://img.shields.io/npm/dm/%40code-pushup%2Fdoc-coverage-plugin)](https://npmtrends.com/@code-pushup/doc-coverage-plugin)
+[![dependencies](https://img.shields.io/librariesio/release/npm/%40code-pushup%2Fdoc-coverage-plugin)](https://www.npmjs.com/package/@code-pushup/doc-coverage-plugin?activeTab=dependencies)
+
+📚 **Code PushUp plugin for tracking documentation coverage.** 📝
+
+This plugin allows you to measure and track documentation coverage in your TypeScript/JavaScript project.
+It analyzes your codebase and checks for documentation on different code elements like classes, functions, interfaces, types, and variables.
+
+Measured documentation types are mapped to Code PushUp audits in the following way:
+
+- The value is in range 0-100 and represents the documentation coverage for all passed results (_documented / total_)
```

Comment:
```suggestion
- `value`: The value is the number of  undocumented nodes -> 4
- `displayValue`: `${value} undocumented ${type}` -> 4 undocumented functions
- `score`: 0.5 -> total nodes 8 undocumented 4 -> 8/4
```

Somethings like this would be better